src/command_modules/azure-cli-storage/tests/test_storage_account_scenarios.py

```diff
@@ -2,13 +2,27 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
+import unittest
 
 from azure.cli.testsdk import (ScenarioTest, JMESPathCheck, ResourceGroupPreparer,
-                               StorageAccountPreparer)
+                               StorageAccountPreparer, api_version_constraint)
 from .storage_test_util import StorageScenarioMixin
+from azure.cli.core.profiles import ResourceType
 
 
+@api_version_constraint(ResourceType.MGMT_STORAGE, max_api='2015-06-15')
+class StorageAccountTests2(StorageScenarioMixin, ScenarioTest):
```

Comment:
Make the constraint decorator applicable to both method and class. The closer decorator is to the method, the more power it has to override the farther decorators.